app/views/map/_filter.html.haml

```diff
@@ -36,7 +36,7 @@
       input_html: { value: @filter_selected_las.id, class: 'select2-ajax', data: { id: @filter_selected_las.id, text: @filter_selected_las.lau_nazev, url: '/local_administration_units/options.json', placeholder: 'např.: Olomouc' } }
     = f.input :tags_filter, label: 'Kategorie', required: false,
       input_html: { value: @tags_filter, class: 'select2-ajax', data: { url: '/categories/options.json', minimumInputLength: 0, multiple: true,
-      init: @tags_filter_values } }
+      init: @tags_filter_values, placeholder: 'Vyberte kategorie' } }
```

Comment:
urcite by tam melo neco byt. Ale premyslim, zda tam nedat spis priklad, nez aby tam bylo: `Kategorie: vyberte kategorii`

..nebo 'vyber typ dokumentu prodej, drazba, ....' ale to je moc dlouhe.